### DISCUSSION TITLE
Change default background color to dark grey

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26050] [Ideas]
IMO change it to dark grey. I know osu! has a theme of pink, but it's already very present in other stuff. Having a completely neutral color for background more enjoyable to look at and looks more modern too. 

You technically "get tired" of looking at the same NON-neutral color the whole time - same goes for website.
NON-neutral colors are good for categories you don't visit as often like chats.

![osu_2023-12-19_14-23-23_copy](https://github.com/ppy/osu/assets/32525896/7b895138-8d80-4a7a-808c-643ef7930dc1)

If you really wanted to keep it as default, you could add changing background color as an supporter-only feature

**Comment by bdach** [comment]
Not happening.
